test/infrastructure/inmemory/main.go

```diff
@@ -51,6 +51,7 @@ import (
 	"sigs.k8s.io/cluster-api/test/infrastructure/inmemory/internal/cloud"
 	cloudv1 "sigs.k8s.io/cluster-api/test/infrastructure/inmemory/internal/cloud/api/v1alpha1"
 	"sigs.k8s.io/cluster-api/test/infrastructure/inmemory/internal/server"
+	"sigs.k8s.io/cluster-api/test/infrastructure/inmemory/internal/webhooks"
```

Comment:
Let's also add the alias.go in inmemory/webhooks and use it here

Just for consistency

Reply:
I think this change is missing